batch/sql/estimated-current.sql

```diff
@@ -193,6 +193,7 @@ DROP TABLE IF EXISTS `job_groups`;
 CREATE TABLE IF NOT EXISTS `job_groups` (
   `batch_id` BIGINT NOT NULL,
   `job_group_id` INT NOT NULL,
+  `update_id` INT DEFAULT NULL,
```

Comment:
Hmm. I don't love that we're adding a bunch of NULLs. This feels like a bit of a footgun.

Reply:
So I just checked the database and I think luckily the update_id can just be 1 rather than NULL. It looks like we fully backwards populated that table:

```
mysql> select * from batch_updates order by batch_id ASC limit 10;
+----------+-----------+-------+--------------+--------+-----------+---------------+----------------+
| batch_id | update_id | token | start_job_id | n_jobs | committed | time_created  | time_committed |
+----------+-----------+-------+--------------+--------+-----------+---------------+----------------+
|        1 |         1 | NULL  |            1 |     59 |         1 | 1578790132304 |           NULL |
|        2 |         1 | NULL  |            1 |     59 |         1 | 1578790397503 |           NULL |
|        3 |         1 | NULL  |            1 |     59 |         1 | 1578791736119 |           NULL |
|        4 |         1 | NULL  |            1 |     59 |         1 | 1578791809858 |           NULL |
|        5 |         1 | NULL  |            1 |     70 |         1 | 1578792081649 |           NULL |
|        6 |         1 | NULL  |            1 |    106 |         1 | 1578792085096 |           NULL |
|        7 |         1 | NULL  |            1 |    106 |         1 | 1578792086147 |           NULL |
|        8 |         1 | NULL  |            1 |    106 |         1 | 1578792098107 |           NULL |
|        9 |         1 | NULL  |            1 |    106 |         1 | 1578792104817 |           NULL |
|       10 |         1 | NULL  |            1 |    106 |         1 | 1578792108769 |           NULL |
+----------+-----------+-------+--------------+--------+-----------+---------------+----------------+
10 rows in set (0.02 sec)
```

Reply:
OK, nice. If we do DEFAULT 1, should this all work as expected?

Reply:
Ok. Unfortunately, any batch with 0 jobs does not have a corresponding batch update...

Reply:
The only job group that would have a NULL update id would be the root job group. Probably the proper way to fix this if a NULL is not okay is to insert a record of update_id = 1 for every batch that is missing in the batch_updates table and do another migration.

Reply:
I think NULL is probably better honestly. the root job group isn't really part of an update.